subsys/bluetooth/controller/Kconfig

```diff
@@ -5,7 +5,6 @@
 #
 
 DT_PATH_NORDIC_RADIO := $(dt_nodelabel_path,radio)
```

Comment:
`DT_PATH_NORDIC_RADIO` should also be removed

Reply:
Done